src/main/java/net/minecraftforge/network/filters/VanillaConnectionNetworkFilter.java

```diff
@@ -42,8 +45,10 @@
 @ChannelHandler.Sharable
 public class VanillaConnectionNetworkFilter extends VanillaPacketFilter
 {
-
     private static final Logger LOGGER = LogManager.getLogger();
+    private static final Set<ResourceLocation> FORGE_ADDED_REGISTRIES = Stream.of(ForgeRegistries.Keys.DATA_SERIALIZERS, ForgeRegistries.Keys.LOOT_MODIFIER_SERIALIZERS, ForgeRegistries.Keys.WORLD_TYPES)
+            .map(ResourceKey::location)
+            .collect(Collectors.toUnmodifiableSet());
```

Comment:
This set's existence should likely be noted in `ForgeRegistries`, for ease of maintenance should a new Forge-exclusive registry be added.